__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest");
+const app = require("../db/app.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+
+beforeAll(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  describe("GET", () => {
+    test("GET 200: responds with status 200 and an array of topic objects", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body: { topics } }) => {
+          expect(topics.length).toBe(3);
+          expect(Array.isArray(topics)).toBe(true);
```

Comment:
We do not need to check if it's an array as that happens implicitly when we use forEach on it :)